bin/main.rs

```diff
@@ -75,6 +102,47 @@ struct Args {
         env = "WADM_MAX_JOBS"
     )]
     max_jobs: usize,
+
+    /// URL of the Nats Jetstream KV that will be used for storage
+    #[arg(long = "storage-nkv-url", env = "STORAGE_NATSKV_URL")]
+    storage_nkv_url: Option<String>,
+
+    /// Optional prefix for the server tate that will be stored.
+    /// this prefix will be prepended to the bucket name of the lattice
+    /// (ex. <prefix>_lattice_<lattice id>).
+    #[arg(
+        long = "storage-nkv-lattice-bucket-prefix",
+        env = "STORAGE_NATSKV_LATTICE_BUCKET_PREFIX"
```

Comment:
Generally environment vars should be prefaced with the application name (i.e. `WADM_`)